test/test_quantization.py

```diff
@@ -369,6 +368,19 @@ def test_fuse_module_eval(self):
         self.assertEqual(type(testMod.sub2.bn), torch.nn.BatchNorm2d,
                          "Non-fused submodule BN")
 
+class ObserverTest(QuantizationTestCase):
```

Comment:
Can we check that the implementation here is identical to the caffe2 implementation?

Reply:
The one I have added matches caffe2. Is there a way to call caffe2 implementation from here?